src/components/button/Button.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const Button = ({
+  children,
+  secondary,
```

Comment:
From what I've found is there always tends to be more variants than just primary and secondary, you end up with ghost, error etc. So could be better to pass a string of `type` and then adding that modifier on